_posts/2019-10-24-this-week-in-glean.md

```diff
@@ -0,0 +1,60 @@
+---
+permalink: "/{{ year }}/{{ month }}/{{ day }}/this-week-in-glean"
+title: "This Week in Glean: A Release"
+published_date: "2019-10-24 17:30:00 +0200"
+layout: post.liquid
+data:
+  route: blog
+  tags:
+    - mozilla
+    - rust
+---
+
+(“This Week in Glean” is a series of blog posts that the Glean Team at Mozilla is using to try to communicate better about our work. They could be release notes, documentation, hopes, dreams, or whatever: so long as it is inspired by Glean.)
+
+Last week's blog post: [This Week in Glean: Glean on Desktop (Project FOG)](https://chuttenblog.wordpress.com/2019/10/17/this-week-in-glean-glean-on-desktop-project-fog/) by chutten.
+
+---
+
+Back in June when [Firefox Preview shipped](https://blog.mozilla.org/futurereleases/2019/06/27/reinventing-firefox-for-android-a-preview/), it also shipped with Glean, our new solution for Telemetry on mobile.
```

Comment:
"shipped with the Glean SDK, our new Telemetry library, initially targetting mobile platforms" (let's leave the mobile part out)